packages/next-swc/crates/next-dev-tests/tests/integration/next/image/basic/input/pages/index.js

```diff
@@ -13,26 +14,51 @@ export default function Home() {
       id="imported"
       alt="test imported image"
       src={img}
-      width="100"
-      height="100"
+      placeholder="blur"
     />,
     <Image
       id="local"
       alt="test src image"
       src="/triangle-black.png"
-      width="100"
+      width="116"
       height="100"
     />,
   ]
 }
 
+console.log(img)
```

Comment:
Should the log be removed?

Reply:
It can stay. It's in the test case and will only be shown in case of errors, so it's helpful in that case.